misc/python/materialize/checks/multiple_partitions.py

```diff
@@ -0,0 +1,103 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+from textwrap import dedent
+from typing import List
+
+from materialize.checks.actions import Testdrive
+from materialize.checks.checks import Check
+
+
+def schemas() -> str:
+    return dedent(
+        """
+       $ set keyschema={
+           "type": "record",
+           "name": "Key",
+           "fields": [
+               {"name": "key1", "type": "string"}
+           ]
+         }
+
+       $ set schema={
+           "type" : "record",
+           "name" : "test",
+           "fields" : [
+               {"name":"f1", "type":"string"}
+           ]
+         }
+    """
+    )
+
+
+class MultiplePartitions(Check):
+    """Test that adds new partitions to a Kafka source"""
+
+    def initialize(self) -> Testdrive:
+        return Testdrive(
+            schemas()
+            + dedent(
+                """
+                $ kafka-create-topic topic=movies-topic
```

Comment:
Can you replace `movies` file-wide with `multiple-partitions` (with a dash in case of a topic name, underscore in case of source and view names).

Reply:
Agreed, makes it easier to know where a failure originated